### ISSUE TITLE
Timeout in ui with --host=0.0.0.0

### ISSUE THREAD

**Issue by sam-s** [mlflow/mlflow#885]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: no
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: aws linux 4.0
- **MLflow installed from (source or binary)**: `pip3 install mlflow`
- **MLflow version (run ``mlflow --version``)**: `0.8.2`
- **Python version**: `3.6.5`
- **npm version (if running the dev UI):** not using npm
- **Exact command to reproduce**: `mlflow ui --host=0.0.0.0`

### Describe the problem

When I run `mlflow ui`, I can using `mlflow` web ui at `http://127.0.0.1:5000` just fine (but only locally).

When I run `mlflow ui --host=0.0.0.0`, I can _connect_ to the port:
```
$ nc -vz localhost 5000
Connection to localhost 5000 port [tcp/commplex-main] succeeded!
```

but the http connection from the local host becomes extremely slow (and remote times out).

### Source code / logs

```
[2019-02-08 18:35:17 +0000] [96834] [INFO] Starting gunicorn 19.9.0
[2019-02-08 18:35:17 +0000] [96834] [INFO] Listening at: http://0.0.0.0:5000 (96834)
[2019-02-08 18:35:17 +0000] [96834] [INFO] Using worker: sync
[2019-02-08 18:35:17 +0000] [96837] [INFO] Booting worker with pid: 96837
[2019-02-08 18:35:49 +0000] [96834] [CRITICAL] WORKER TIMEOUT (pid:96837)
[2019-02-08 18:35:49 +0000] [96837] [INFO] Worker exiting (pid: 96837)
[2019-02-08 18:35:49 +0000] [97029] [INFO] Booting worker with pid: 97029
```

**Comment by DanlanChen** [comment]
i have the same error, did you solve it @sam-s

**Comment by jakubczakon** [comment]
@sam-s @DanlanChen 
You could also have your MLflow projects hosted and managed in a lightweight [neptune.ml][1] knowledge sharing hub. 

There are two benefits to it:

 - you can avoid all the problems with MLflow's UI,
 - you can share your experiment runs with everyone. For example, [check out this project][2]. 

I know it sounds salesly but Neptune is actually free (for non-organizations) and I really feel that this integration brings a lot of value to the MLflow community.

Anyhow, I hope this helps.

Also if you are interested, I have just written a [blog post about it][3].

  [1]: https://neptune.ml/?utm_source=stack_exchange&utm_medium=forum&utm_campaign=neptune_is_free&utm_term=mlflow
  [2]: https://ui.neptune.ml/jakub-czakon/mlflow-integration/experiments
  [3]: https://medium.com/neptune-ml/collaborate-on-mlflow-experiments-in-neptune-fb4f8f84a995

**Comment by louisza** [comment]
I think this is solved in issue #101

**Comment by sameermahajan** [comment]
mlflow (version 1.0.0) does not seem to support

--host=0.0.0.0

option for ui command. So this issue is no longer applicable.
